### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by ghost** [victornpb/undiscord#616] [1. Questions & Answers]

**Comment by FrAmed46** [comment]
i think you can with the data packet. I havent tried it yet but i just got banned so i will try. Just request your data packet and there is a field where you put in undiscord

**Reply by SpiritAxolotl** [reply]
nope.
```
Error searching messages, API responded with status 403!
	{"message":"Missing Access","code":50001}
CoreException	{}
```

**Comment by ghost** [comment]
Doubt it. Even deleting your entire account leaves the content preserved under a Deleted User

**Reply by yoshimo** [reply]
That sounds like a major issue with the GDPR in Europe and might be actionable.
